test/driver.cpp

```diff
@@ -23,66 +23,83 @@
 
 #include "tester.h"
 
+using std::string;
+using std::string_view;
+using std::size_t;
+
 //must be defined in a unit-specific source file such as "array-test.cpp"
 void runTests();
 
-void processCmdLine(std::vector<std::string_view> arguments);
+void processCmdLine(char* arguments[], const size_t size);
+
+void replace_all(string& str, const string_view substr, 
+   const string& toReplace);
```

Comment:
I propose the following function declaration to go with the suggested change to the function's definition:

```suggestion
void replace_all(std::string& str, const std::string& substr, const std::string& new_substr);
```

Reply:
@smruthys I have made the changes and pushed to origin. Thanks for the feedback.